spoc/io.py

```diff
@@ -1,67 +1,98 @@
 """Persisting functionality of spoc that manages writing to and reading from the filesystem."""
 
 import pickle
-from typing import Dict, Union
+from typing import Dict, Union, List
+from hashlib import md5
+import os
+import json
+from pathlib import Path
 import pandas as pd
 import dask.dataframe as dd
-from .contacts import Contacts
-from .pixels import Pixels
-from .dataframe_models import FragmentSchema, ContactSchema
-from .fragments import Fragments
+from spoc.contacts import Contacts
+from spoc.pixels import Pixels
+from spoc.models.file_parameter_models import ContactsParameters, PixelParameters
+from spoc.fragments import Fragments
 
 
 class FileManager:
```

Comment:
For now it seems to represent a bunch of IO operations. In terms of my proposal document, it can actually represent the LargeDirectory.